yml/dp02_dc2.yaml

```diff
@@ -9044,6 +9064,7 @@ tables:
     description: Number of columns in the image.
     mysql:datatype: INTEGER
     fits:tunit: pixel
+    ivoa:ucd: meta.number;instr.det  # FIXME: Not sure about instr.det
```

Comment:
`meta.number;instr.det` is good.  It doesn't have to be identical to ObsCore `s_xel1/2` which is just `meta.number`.  I think what we're doing is better.